mysql_example/mysql_example.py

```diff
@@ -276,7 +277,8 @@ def dbWriter(sql, rows) :
 c.execute("CREATE TABLE plural_block "
           "(SELECT block_id, donor_id "
           " FROM blocking_map INNER JOIN plural_key "
-          " USING (block_key))")
+          " USING (block_key)"
+          " GROUP BY block_key, donor_id)")
```

Comment:
This really should not be happening. Would you mind seeing if you can reproduce and posting the predicates that dedupe had learned on that run?